bmap-rs/Cargo.toml

```diff
@@ -12,3 +12,8 @@ anyhow = "1"
 structopt = "0.3"
 nix = "0.25"
 flate2 = "1"
+hyper = { version = "0", features = ["full"] }
```

Comment:
since this file is managed by Dependabot, use full versions for these dependencies